config/contributors.ts

```diff
@@ -6,11 +6,11 @@ export const Contributor: IContributor[] = [
     role: 'Frontend Developer',
     contributions: [
       {
-        title: 'add show/hide password feature for sign-in and login page',
```

Comment:
/create a new list item with the details .. leave the previous one as before :)

also am able to see that you haven't pulled the code. Kindly pull the code to resolve merge conflicts